st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -84,17 +83,10 @@ def _set_up_logger(self):
         Set up a logger which logs all the messages with level DEBUG
         and above to stderr.
         """
+        logging.config.fileConfig('/etc/st2actions/logging.conf')
```

Comment:
I just had a look at the code, it looks like you should be able to use the existing configuration option - `cfg.CONF.actionrunner.logging`